src/schemas/video.js

```diff
@@ -12,51 +11,49 @@ export default {
 		},
 		parseDOM: [
 			{
-				tag: 'video',
+				tag: 'figure',
 				getAttrs: (node) => {
+					if (node.getAttribute('data-node-type') !== 'video') {
```

Comment:
If `data-node-type` mostly exists to specify the type of node that the `<figure>` contains, it might be a little cleaner to just check `node.firstChild.tagName` — just one less similarly-named field that needs to match for things to work.

Reply:
I'd prefer this if everything could use this technique. But for things like equations, which are nested spans, I prefer the explicit consistency of `data-node-type`. I sympathize though, that some of this does get a bit verbose.